Files/UserControls/Selection/RawSelectedItems.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Collections;
+
+namespace Files.UserControls.Selection
+{
+    public class RawSelectedItems : ISelectedItems
```

Comment:
What's the difference with GenericSelectedItems?

Reply:
and of this :blush: